### PR TITLE
add sample files for Sql/mysql/postgresql

### PR DESCRIPTION
**Pull request by yungezz**
## Description
sample file naming convention is
```
<target>-<authType>-<clientType>.language_file_extension.
```

At portal, will construct the sample file github url based on the naming convention.
- If github file with convention name not found,  will fall back to clientType python.
- If github file with python still not found, will fall back to root folder of the github repo

https://msazure.visualstudio.com/One/_git/ServiceLinker-PortalExtension?path=/src/Default/ServiceLinkerExtension/Client/React/Shared/SampleHelper.ts&_a=contents&version=GBmaster

### CODE REVIEW

File: src/sqldatabase-systemidentity-nodejs.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const sql = require('mssql');
+
+async function main() {
+    const host = process.env.AZURE_SQL_SERVER;
+    const database = process.env.AZURE_SQL_DATABASE;
+    const username = process.env.AZURE_SQL_USERNAME;
+    const password = process.env.AZURE_SQL_PASSWORD;
```
Comment (xfz11) [inline src/sqldatabase-systemidentity-nodejs.js:7]: no `password` for system mi

File: src/mysql-secret-django.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': database,
+        'USER': user,
+        'PASSWORD': password,
+        'HOST': host,
+        'PORT': port
+    }
+}
```
Comment (houk-ms) [inline src/mysql-secret-django.py:19]: why no client?

File: src/postgresql-systemassignedidentity-dotnet.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            var connectionString = Environment.GetEnvironmentVariable("AZURE_POSTGRESQL_CONNECTIONSTRING");
+            var tokenCredential = new DefaultAzureCredential();
+            var accessToken = await tokenCredential.GetTokenAsync(new Azure.Core.TokenRequestContext(new[] { "https://ossrdbms-aad.database.windows.net/.default" }));
+            var builder = new NpgsqlConnectionStringBuilder(connectionString)
+            {
+                SslMode = SslMode.Require,
+                TrustServerCertificate = true,
+                IntegratedSecurity = true,
+                Username = new NpgsqlConnectionStringBuilder($"user=<{accessToken.Token}>").Username,
```
Comment (xfz11) [inline src/postgresql-systemassignedidentity-dotnet.cs:20]: Username should have been set in AZURE_POSTGRESQL_CONNECTIONSTRING. Does the code work?

File: src/postgresql-systemassignedidentity-java.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import java.sql.SQLException;
+
+public class Main {
+    public static void main(String[] args) {
+        String connectionString = System.getenv("AZURE_POSTGRESQL_CONNECTIONSTRING");
+        
+        try {
+            DefaultAzureCredential tokenCredential = new DefaultAzureCredential();
+            String accessToken = tokenCredential.getToken("https://ossrdbms-aad.database.windows.net/.default").getToken();
+            String url = String.format("%s?sslfactory=org.postgresql.ssl.NonValidatingFactory&authenticationPluginClassName=com.azure.identity.extensions.jdbc.postgresql.AzurePostgresqlAuthenticationPlugin", connectionString);
```
Comment (xfz11) [inline src/postgresql-systemassignedidentity-java.java:15]: Why we use `sslfactory=org.postgresql.ssl.NonValidatingFactory`

File: src/sqldatabase-systemidentity-dotnet.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
+using System.Data.SqlClient;
+using System;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string connString = Environment.GetEnvironmentVariable("AZURE_SQL_CONNECTIONSTRING");
```
Comment (houk-ms) [inline src/sqldatabase-systemidentity-dotnet.cs:10]: why system identity is using connection string?

File: src/mysql-secret-django.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import os
+import mysql.connector
+
```
Comment (houk-ms) [inline src/mysql-secret-django.py:3]: how about adding some comments telling users how to find these env variable names

File: src/sqldatabase-systemidentity-python.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import pyodbc
+import os
+
+host = os.environ['AZURE_SQL_HOST']
+database = os.environ['AZURE_SQL_DATABASE']
+username = os.environ['AZURE_SQL_USERNAME']
+password = os.environ['AZURE_SQL_PASSWORD']
```
Comment (xfz11) [inline src/sqldatabase-systemidentity-python.py:7]: no password

File: src/sqldatabase-systemidentity-dotnet.cs (hunk 5 above)
Comment (yungezz) [reply src/sqldatabase-systemidentity-dotnet.cs:10]: https://review.learn.microsoft.com/en-us/azure/service-connector/tutorial-passwordless?branch=pr-en-us-237756&tabs=system&pivots=sql#connect-to-database-with-azure-active-directory-authentication

File: src/postgresql-systemassignedidentity-java.java (hunk 4 above)
Comment (yungezz) [reply src/postgresql-systemassignedidentity-java.java:15]: copied from other code samples

**Pull request closed without merging**